registry.json

```diff
@@ -5096,5 +5096,18 @@
    "telegram": "wafflepool",
    "operatorName": "",
    "github": ""
- }
+ },
+ {
+	"ticker": "SAPIO",
+	"poolId": "c3cd2527a5f0b82a86ff69fdd5a3440dc51bcb957953ba19137738c0",
+	"social": {
+		"twitter": "nikos_SAPIO",
+		"facebook": "",
+		"instagram": "",
+		"youtube": "https://www.youtube.com/channel/UCcPH2RMsszRGJ2awvLdMKzQ"
+	},
+	"telegram": "Sapiopool",
```

Comment:
Please change this to be the telegram username of an individual and not a group or channel